code/datums/components/combustible_flooder.dm

```diff
@@ -0,0 +1,56 @@
+/datum/component/combustible_flooder
+	// Gas type, molar count, and temperature. All self explanatory.
+	var/gas_name
```

Comment:
name? or id

Reply:
it uses the gas id, so water vapor would be `water_vapor` rather then the name, which is used for display `Water Vapor`
![image](https://user-images.githubusercontent.com/58055496/129115139-a8aea9b8-5185-49f1-b36f-110481c031a5.png)